index.html

```diff
@@ -9,90 +9,97 @@
+          Node.js is a JavaScript runtime built on Chrome's V8 JavaScript engine. Uses an event-driven, non-blocking I/O model taht makes it lightweight and efficient.
+        </p>
+      </div>
+      <div class="tech-container">
+        <img class="tech-logo" src="assets/aspnet_icon.png" alt="Asp. NET logo">
+        <p class="tech-title">ASP.NET Core</p>
+        <p class="tech-sub">Release: 2016</p>
+        <p class="tech-desc">
+          ASP.NET Core is the next generation of ASP.NET. It is a modular framework that runs on both the full .NET Framework, on Windows, and the cross-platform .NET Core.
+        </p>
+      </div>
+      <div class="tech-container">
+        <img class="tech-logo" src="assets/django_icon.svg" alt="DJango logo">
+        <p class="tech-title">Django</p>
+        <p class="tech-sub">Release: 2005</p>
+        <p class="tech-desc">
+          Django (/ˈdʒæŋɡoʊ/ JANG-goh)[5] is a free and open-source web framework, written in Python, which follows the model-view-template (MVT) architectural pattern.
+        </p>
+      </div>
     </div>
   </div>
-  <h2 class="page-title">Back-End Tools</h2>
-  <div class="tech-section">
-    <div class="tech-container">
-      <img class="tech-logo" src="assets/rails_icon.png" alt="Rails logo">
-      <p class="tech-title">Ruby on Rails</p>
-      <p class="tech-sub">Release: 2005</p>
-      <p class="tech-desc">
-        Ruby on Rails, or Rails, is a server-side web application framework written in Ruby. Rails is an MVC framework, providing default structures for a database, a web service, and web pages.
-      </p>
-    </div>
-    <div class="tech-container">
-      <img class="tech-logo" src="assets/express_icon.svg" alt="NodeJS logo">
-      <p class="tech-title">NodeJS</p>
-      <p class="tech-sub">Release: 2009</p>
-      <p class="tech-desc">
-        Node.js is a JavaScript runtime built on Chrome's V8 JavaScript engine. Uses an event-driven, non-blocking I/O model taht makes it lightweight and efficient.
-      </p>
-    </div>
-    <div class="tech-container">
-      <img class="tech-logo" src="assets/aspnet_icon.png" alt="Asp. NET logo">
-      <p class="tech-title">ASP.NET Core</p>
-      <p class="tech-sub">Release: 2016</p>
-      <p class="tech-desc">
-        ASP.NET Core is the next generation of ASP.NET. It is a modular framework that runs on both the full .NET Framework, on Windows, and the cross-platform .NET Core.
-      </p>
-    </div>
-    <div class="tech-container">
-      <img class="tech-logo" src="assets/django_icon.svg" alt="DJango logo">
-      <p class="tech-title">Django</p>
-      <p class="tech-sub">Release: 2005</p>
-      <p class="tech-desc">
-        Django (/ˈdʒæŋɡoʊ/ JANG-goh)[5] is a free and open-source web framework, written in Python, which follows the model-view-template (MVT) architectural pattern.
-      </p>
-    </div>
+  <div class="footer-container">
+    <p class="footer-title">We are game changers</p>
```

Comment:
This is supposed to be quite a big text isnt it? Why dont you use something like <h3> or <h4> ?